ci/test_python_singlegpu.sh

```diff
@@ -19,6 +19,21 @@ pytest \
   --cov=cuml \
   --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-coverage.xml" \
   --cov-report=term \
+  -m "not memleak" \
+  .
+
+  rapids-logger "memory leak pytests..."
+
+pytest \
+  --numprocesses=1 \
+  --ignore=dask \
+  --cache-clear \
+  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml.xml" \
+  --cov-config=../../.coveragerc \
+  --cov=cuml \
+  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-coverage.xml" \
```

Comment:
```suggestion
  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml-memleak.xml" \
  --cov-config=../../.coveragerc \
  --cov=cuml \
  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-memleak-coverage.xml" \
```

I think the `junitxml` and `cov-report` flags should have values that are different from the `pytest` invocation above this one.